### PR TITLE
Updated Documentation format for teams cmdlets

### PR DESCRIPTION
**Pull request by bkalyan976**
Updated Documentation format for teams cmdlets Get-CsTeamsFirstPartyMeetingTemplateConfiguration, New-CsTeamsMeetingTemplatePermissionPolicy, Remove-CsTeamsMeetingTemplatePermissionPolicy, Get-CsTeamsMeetingTemplateConfiguration

### CODE REVIEW

**Review by get-itips — APPROVED** [review]
I'm ok after @tseward suggestion

File: teams/teams-ps/teams/New-CsTeamsMeetingTemplatePermissionPolicy.md [hunk 1]
```diff
@@ -27,21 +27,22 @@ Creates a new instance of the policy with a name and a list of hidden meeting te
...
-```powershell
-PS> $hiddentemplate_1 = New-CsTeamsHiddenMeetingTemplate -Id customtemplate_9ab0014a-bba4-4ad6-b816-0b42104b5056
-PS> $hiddentemplate_2 = New-CsTeamsHiddenMeetingTemplate -Id firstparty_e514e598-fba6-4e1f-b8b3-138dd3bca748
-```
-
 The next step would be to create the policy instance.
-
 ```powershell
-PS> New-CsTeamsMeetingTemplatePermissionPolicy -Identity Test_Policy -HiddenMeetingTemplates @($hiddentemplate_1, $hiddentemplate_2) -Description "This is a test policy"
+$hiddentemplate_1 = New-CsTeamsHiddenMeetingTemplate -Id customtemplate_9ab0014a-bba4-4ad6-b816-0b42104b5056
```
[the remark this answers is not in the document]
Comment (bkalyan976) [reply teams/teams-ps/teams/New-CsTeamsMeetingTemplatePermissionPolicy.md:32]: its not required. REF: https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.3#illustrative-examples:~:text=%60%60%60powershell%0AGet%2DCommand%20%2DModule%20Microsoft.PowerShell.Security%0A%60%60%60


I'm following same structure as mentioned in above reference.

if we add PS > its breaking existing scripts

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review - Just @tseward's suggestion.

**Review by dariomws — APPROVED** [review]
@bkalyan976, we usually find the description of the examples after the code and the output.

**Comment by yogkumgit** [comment]
@bkalyan976 Please implement the suggested changes. Thanks

File: teams/teams-ps/teams/New-CsTeamsMeetingTemplatePermissionPolicy.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bkalyan976) [reply teams/teams-ps/teams/New-CsTeamsMeetingTemplatePermissionPolicy.md:32]: @tseward  could you check this and approve please, this is breaking our existing scripts

**Comment by yogkumgit** [comment]
@tseward

**Pull request merged**